publication-commons/src/main/java/no/unit/nva/publication/service/impl/ReadResourceService.java

```diff
@@ -149,7 +149,8 @@ private QueryRequest queryByResourceIndex(ResourceDao queryObject) {
         return new QueryRequest()
                    .withTableName(tableName)
                    .withIndexName(BY_CUSTOMER_RESOURCE_INDEX_NAME)
-                   .withKeyConditions(keyConditions);
+                   .withKeyConditions(keyConditions)
+                   .withLimit(100);
```

Comment:
The default is 50, which is too low right now.

A new PR will split the drafts/published into different queries and allow pagination of results